MSAL/test/integration/native_auth/end_to_end/sign_in/MSALNativeAuthSignInUserNameAndPasswordEndToEndTests.swift

```diff
@@ -63,21 +63,17 @@ final class MSALNativeAuthSignInUsernameAndPasswordEndToEndTests: MSALNativeAuth
 
     // Hero Scenario 2.2.1. Sign in – Email and Password on SINGLE screen (Email & Password)
     func test_signInUsingPasswordWithKnownUsernameResultsInSuccess() async throws {
-        throw XCTSkip("Skipping this test because native auth KeyVault is missing")
-        guard let sut = initialisePublicClientApplication() else {
+        guard let sut = initialisePublicClientApplication(), let username = getSignInUsernamePassword(), let password = await retrievePasswordForSignInUsername() else {
             XCTFail("Missing information")
             return
         }
 
         let signInExpectation = expectation(description: "signing in")
         let signInDelegateSpy = SignInPasswordStartDelegateSpy(expectation: signInExpectation)
 
-        let username = ProcessInfo.processInfo.environment["existingPasswordUserEmail"] ?? "<existingPasswordUserEmail not set>"
-        let password = ProcessInfo.processInfo.environment["existingUserPassword"] ?? "<existingUserPassword not set>"
-
         sut.signIn(username: username, password: password, correlationId: correlationId, delegate: signInDelegateSpy)
 
-        await fulfillment(of: [signInExpectation], timeout: 2)
+        await fulfillment(of: [signInExpectation], timeout: defaultTimeout)
```

Comment:
Since we use this expression in every test and we almost always use `defaultTimeout`, maybe we can create a custom `fulfillment()` function in `MSALNativeAuthEndToEndBaseTestCase` that takes `defaultTimeout` as a default parameter.

```
    public func fulfillment(
        of expectations: [XCTestExpectation],
        timeout seconds: TimeInterval = MSALNativeAuthEndToEndBaseTestCase.defaultTimeout
    ) async {
        await fulfillment(of: expectations, timeout: seconds, enforceOrder: false)
    }
```

Reply:
Good idea. I couldn't use the `defaultTimeout`. I had to make it static, and I didn't like it. So, I removed the defaultTimeout attribute and used a default value in the method (default timeout)